plugins/coatils.py

```diff
@@ -163,7 +165,8 @@ def position(stl, stc, enl, enc):
                                                               enl, enc)
 
     # Ignore PycodestyleBear, LineLengthBear
-    @re_botcmd(pattern=r'^run\s+(\w+)((?:\s+\w+(?:\s+\w+=\w+)*)+)\n+```\n([\s\S]+)\n```$')
+    @re_botcmd(pattern=r'^run\s+(\w+)((?:\s+\w+(?:\s+\w+=\w+)*)+)\n+```\n([\s\S]+)\n```$',
+               re_cmd_name_help='run <Bear [[set=ing]+]>+\n```\n<code>+\n```')
```

Comment:
to separate the setting name from the setting value, consider it as a delimiter. Similar to what we have in coafiles